src/sass/style.scss

```diff
@@ -64,3 +64,31 @@ body {
 .grav-c-two-columns-text {
   margin: 2rem auto 0em auto;
 }
+
+// Temporary overrides - About page partner logos
+.grav-c-list-inline-row {
+  margin-bottom: $grav-sp-m;
+  justify-content: center;
+}
+
+.grav-c-list-inline-row>li {
+  flex: auto 0 0;
+  margin-right: $grav-sp-m;
+  margin-left: $grav-sp-m;
+}
+
+.grav-c-list-inline-row a {
+  height: auto;
+  border-width: 1px;
+  border-color: #DDDFDF;
```

Comment:
Can you use one of Gravity's `$grav-co-neutral-...` colour variables here? Or else, one of the `$color-grey-...` ones the flourishes use (defined in `_colours.scss`). I think it's better to reuse existing colours where we can (preferably ones from our design system) rather than introducing new ones.

Reply:
Sure : `$color-grey-iron` seemed to do the job